src/fallbacks.jl

```diff
@@ -111,7 +111,7 @@ end
 coordinates(t::AbstractFeatureCollectionTrait, fc) = map(f -> coordinates(f), getfeature(t, fc))
 
 extent(::Any, x) = Extents.extent(x)
-function calc_extent(t::AbstractPointTrait, geom)
+function calc_extent(::AbstractProjectedTrait, t::AbstractPointTrait, geom)
```

Comment:
Won't this break getting extents for lat/lon?

Reply:
Well yes, but the extents of lat lon are wrong, as in you need to return a geographic extent to make the box useful for intersection etc?

Reply:
If all extents are in lat/lon it's not wrong?

This would be a catastrophically breaking change

Reply:
Do antimeridian crossings work in Extents?

Reply:
Probably not.

Reply:
We could do Cyclic dimensions like DimensionalData.jl has

Reply:
Let's discuss further in https://github.com/rafaqz/Extents.jl/issues/27. As long as no-one implements GeographicTrait in the ecosystem, this is not breaking anything.

Reply:
Ok lets not merge the PR until we have that worked out